weblate/trans/tests/test_remote.py

```diff
@@ -311,7 +329,12 @@ def test_file_scan_commit_policy(self) -> None:
         self.assertEqual(unit_1.state, STATE_TRANSLATED)
```

Comment:
Test fails at L328 because the change is overwritten during file scan. The unit does not match because `original_state` read from the incoming unit in the repository is `STATE_TRANSLATED` whereas the in `unit.details["disk_state"]` it is `STATE_APPROVED`.